web/packages/teleport/src/components/ToolTipNoPermBadge/ToolTipNoPermBadge.tsx

```diff
@@ -71,16 +77,27 @@ export const ToolTipNoPermBadge: React.FC<Props> = ({
           horizontal: 'left',
         }}
       >
-        <StyledOnHover px={3} py={2} data-testid="tooltip-msg">
+        <StyledOnHover
+          px={3}
+          py={2}
+          data-testid="tooltip-msg"
+          modalCss={() => `pointer-events: none`}
```

Comment:
What is `modalCss`? AFAICT `modalCss` is not doing anything in this component. (Though it works in `Popover` because [`Popover` passes it through to a `Modal`](https://github.com/gravitational/teleport/blob/d1664b7f39742a2c2c27b244d0abe9a58878aa06/web/packages/design/src/Popover/Popover.jsx#L323))

Reply:
You are right. Removed in https://github.com/gravitational/teleport/pull/31022/commits/3a71196fe14e220df8686c0085045d1ed19cd07a